gnd/src/main/res/layout/offline_areas_frag.xml

```diff
@@ -1,5 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--
+<?xml version="1.0" encoding="utf-8"?><!--
```

Comment:
\n needed between > and <. Formatter did this to me too, but it will fail checkstyle.

Reply:
Done! Thanks!